src/js/themes/base.d.ts

```diff
@@ -1235,7 +1235,7 @@ export interface ThemeType {
     iconContainer?: BoxProps;
     textContainer?: BoxProps;
     title?: TextProps;
-    message?: TextProps;
+    message?: TextProps | { fill?: boolean };
```

Comment:
Should this be TextProps and fill? instead of TextProps or fill

Reply:
aw yes